src/components/auth/login/Login.jsx

```diff
@@ -1,38 +1,49 @@
 import React, { useState } from "react";
 import { auth } from "../../config/firebaseConfig";
 import db from "../../config/firebaseConfig";
-import AddProducts from "../../addProductsForm/AddProducts";
 import Button from "react-bootstrap/Button";
-import { Col, Row } from "react-bootstrap";
+import { Col } from "react-bootstrap";
 import Form from "react-bootstrap/Form";
+import {
+  logIn,
+  logOut,
+  logInError,
+  popUpStatus,
+} from "../../redux/actions/index";
+import { useSelector, useDispatch } from "react-redux";
 
 const LogInForm = () => {
-  const [isAdmin, setIsAdmin] = useState(false);
-  const [isLoggedIn, setLoggedIn] = useState(false);
+  const { isLoggedIn, authError } = useSelector((state) => state.handleLogin);
```

Comment:
this seems weird, handlelogin is a redux action right? usually thats not on the state, that should be accessed using dispatch